docs/static/monitoring-apis.asciidoc

```diff
@@ -0,0 +1,312 @@
+       java.lang.Thread.sleep(Native Method)
+       org.jboss.netty.util.HashedWheelTimer$Worker.waitForNextTick(HashedWheelTimer.java:445)
+       org.jboss.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:364)
+       org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
+       java.lang.Thread.run(Thread.java:745)
+--------------------------------------------------
+
+/////
+Commenting this section out because it looks like this API was moved to Alpha 2. Feel free to add review
+comments, tho, if you notice inaccuracies.
+
+[[plugins-api]]
+=== Plugins API
+
+experimental[]
+
+The plugins API gets information about all Logstash plugins that are currently installed.
+This API basically returns the output of running the `bin/plugins list --verbose` command.
+
+[source,js]
+--------------------------------------------------
+GET /_plugins
+--------------------------------------------------
+//AUTOSENSE
+
+The output is a JSON document.
+
+Example response:
+
+[source,js]
+--------------------------------------------------
+"total": 102
+"plugins" : [
+  {
+      "name": "logstash-output-pagerduty"
+      "version": "2.0.2"
+  },
+  {
+      "name": "logstash-output-elasticsearch"
+      "version": "2.1.2"
+  }
+....
+] 
+--------------------------------------------------
+/////
+
+[[root-resource-api]]
+=== Root Resource API
+
+experimental[]
+
+//QUESTION: Is "root resource API" a good name for this API? 
+
+The root resource API retrieves general information about the Logstash instance, including
+the host name, version, and build hash.
+
+//QUESTION: Actually I tested the REST call on the latest build, the response did not include the build hash. Is this field still part of the output?
+
+[source,js]
+--------------------------------------------------
+GET /
+--------------------------------------------------
+//AUTOSENSE
+
+Example response:
+
+[source,js]
+--------------------------------------------------
+{
+   "hostname": "skywalker",
+    "version" : {
+        "number" : "2.1.0",
+        "build_hash" : "72cd1f1a3eee09505e036106146dc1949dc5dc87"
+    }
+  }
+--------------------------------------------------
+
+[[stats-info-api]]
+=== Stats Info API
+
+experimental[]
+
+The stats info API retrieves runtime stats about Logstash. 
+
+To retrieve all stats for the Logstash instance, use the `_node/stats` endpoint:
+
+[source,js]
+--------------------------------------------------
+GET /_node/stats/<types>
+--------------------------------------------------
+//AUTOSENSE
+
+To retrieve all stats per pipeline, use the `_pipelines/stats` endpoint:
+
+//QUESTION: Should we mention that for Alpha 1 that the API will only show one pipeline, main? Actually, when I try the REST call, it returns an empty document. Is the _pipelines/stats endpoint even supported in Alpha 1? I'm using a very basic Logstash config. Do I need to configure something to see pipeline stats? 
+
```

Comment:
The stats are dynamically created at runtime, if the pipeline don't do anything no stats will be collected. If you are using the `generator` input to send to elasticsearch you should get some metrics. Can you share you configuration?